Update the Multifactor Policy section of the Policy API document

edjohns-okta opened the pull request:
Resolves: OKTA-99063
Bacon: test

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -890,11 +890,38 @@ The following conditions may be applied to the rules associated with Okta Sign O
 
 
-## MFA Policy
+## Multifactor Policy
```
Add "(MFA)" after Multifactor since we use MFA throughout the doc.

edjohns-okta replied:
👍

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -890,11 +890,38 @@ The following conditions may be applied to the rules associated with Okta Sign O
 > The MFA Policy API is a {% api_lifecycle beta %} [release](/docs/api/getting_started/releases-at-okta.html).
-<!-- [THIS SECTION IN PROGRESS] -->
+
+Multifactor Policy controls which Multifactor Authentication methods (MFA's) are available for a user, as well as when a user may enroll in a particular factor.  
```
If you add "MFA" to title, you can change 898 to "Multifactor (MFA) Policy controls which MFA methods are available..."

edjohns-okta replied:
👍

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -903,53 +930,81 @@ The following conditions may be applied to the rules associated with Okta Sign O
 Parameter | Description | Data Type | Required | Default
 | --- | --- | --- | --- | ---
-self |  | `NOT_ALLOWED`, `OPTIONAL` or `REQUIRED` | Yes | 
+self | Requirements for user initiated enrollment | `NOT_ALLOWED`, `OPTIONAL` or `REQUIRED` | No | `NOT_ALLOWED`
```
"user-initiated"

edjohns-okta replied:
👍

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -903,53 +930,81 @@ The following conditions may be applied to the rules associated with Okta Sign O
+okta_sms | Okta SMS | <a href="#PolicyFactorObject">Policy MFA Factor Object</a> | No | See below  
+rsa_token | RSA Token | <a href="#PolicyFactorObject">Policy MFA Factor Object</a> | No | See below  
+symantec_vip | Symantic VIP | <a href="#PolicyFactorObject">Policy MFA Factor Object</a> | No | See below  
 
```
Do we need the "See below" entries? I'm afraid they'll just be confusing. Okay to leave the default column blank until you get down to the final nested object...

mystiberry-okta replied:
or delete the "Default" column

edjohns-okta replied:
OK, I made the change, let me know how you think it looks.

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -903,53 +930,81 @@ The following conditions may be applied to the rules associated with Okta Sign O
-enroll |  | <a href="#PolicyFactorEnrollObject">Policy Factor Enroll Object</a> | No | 
+consent | Consent requirements for the factor | <a href="#PolicyFactorConsentObject">Policy Factor Consent Object</a> | No | See below
+enroll | Enrollment requirements for the factor | <a href="#PolicyFactorEnrollObject">Policy Factor Enroll Object</a> | No | See below 
 
```
Same comment as above, not sure the See below helps.

edjohns-okta replied:
As above

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -903,53 +930,81 @@ The following conditions may be applied to the rules associated with Okta Sign O
 #### Policy Factor Consent Object
 {: #PolicyFactorConsentObject }
 
+The Policy Factor Consent object is an extensibility point.  In the future policy may be configureable to require user consent to specified terms when enrolling in a factor.   At present settings other than type = `NONE` are ignored.
```
We use a capital letter for Object in other places.

mystiberry-okta replied:
Add comma after "In the future" to prevent 'garden path' effect.

edjohns-okta replied:
Done!

mystiberry-okta commented on _source/_docs/api/resources/policy.md:
```diff
@@ -903,53 +930,81 @@ The following conditions may be applied to the rules associated with Okta Sign O
+
+Parameter | Description | Data Type | Required | Default
+| --- | --- | --- | --- | ---
+terms | Specifies the consent terms to be offered the user upon enrolling in the factor. | <a href="#PolicyFactorConsentTerms">Policy Factor Consent Terms</a> | No | See below
```
Remove See below? okay to leave blank.

edjohns-okta replied:
+1

edjohns-okta commented:
I removed the "Default" columns where you suggested, and I also removed the "See below" items from a few other places, and corrected a couple other minor things.

mystiberry-okta commented:
Passe bacon, merge function has issues, @rchild-okta
